### PR TITLE
NET Interactive: Enable_try_dotnet_to_batch_10a

### PR DESCRIPTION
**Pull request by v-mepa**
**Enable_try_dotnet_to_batch_10a**

https://docs.microsoft.com/en-us/dotnet/api/system.iformatprovider | Batch 10 | Only first example
-- | -- | --
https://docs.microsoft.com/en-us/dotnet/api/system.activator.createinstance | Batch 10 | Only examples 1,2, 6
https://docs.microsoft.com/en-us/dotnet/api/system.array.length | Batch 10 |  
https://docs.microsoft.com/en-us/dotnet/api/system.guid | Batch 10 |  
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.indexof | Batch 10 | Duplicate samples for all   overloads? Potentially move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.sorteddictionary-2 | Batch 10 | Just the sample from the   examples section
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.sortedlist-2 | Batch 10 | Only first example
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.stack-1 | Batch 10 |  
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.reverse | Batch 10 | Duplicate samples for all   overloads? Potentially move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.copyto | Batch 10 | Duplicate samples for all   overloads? Potentially move it to the MemberGroup

### CODE REVIEW

**Comment by v-mepa** [comment]
FYI @mairaw

File: xml/System.Collections.Generic/SortedDictionary`2.xml [hunk 1]
```diff
@@ -127,8 +127,8 @@
   
  Finally, the example demonstrates the <xref:System.Collections.Generic.SortedDictionary%602.Remove%2A> method.  
   
- [!code-csharp[Generic.SortedDictionary#1](~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedDictionary/CS/source.cs#1)]
- [!code-vb[Generic.SortedDictionary#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.SortedDictionary/VB/source.vb#1)]  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedDictionary/CS/source.cs" interactive="try-dotnet" id="Snippet1":::
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.SortedDictionary/VB/source.vb" id="Snippet1":::
```
Comment (mairaw) [inline xml/System.Collections.Generic/SortedDictionary`2.xml:131]: Let's undo this change. There's a docs bug here with the new syntax.

```suggestion
[!code-csharp[Generic.SortedDictionary#1](~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedDictionary/CS/source.cs#1)]
[!code-vb[Generic.SortedDictionary#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.SortedDictionary/VB/source.vb#1)]  
```

File: xml/System.Collections.Generic/SortedList`2.xml [hunk 2]
```diff
@@ -116,9 +116,9 @@
   
  The `foreach` statement of the C# language (`for each` in C++, `For Each` in Visual Basic) returns an object of the type of the elements in the collection. Since the elements of the <xref:System.Collections.Generic.SortedList%602> are key/value pairs, the element type is not the type of the key or the type of the value. Instead, the element type is <xref:System.Collections.Generic.KeyValuePair%602>. For example:  
   
- [!code-cpp[Generic.SortedList#12](~/samples/snippets/cpp/VS_Snippets_CLR/Generic.SortedList/cpp/remarks.cpp#12)]
- [!code-csharp[Generic.SortedList#12](~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedList/CS/remarks.cs#12)]
- [!code-vb[Generic.SortedList#12](~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.SortedList/VB/remarks.vb#12)]  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/Generic.SortedList/cpp/remarks.cpp" id="Snippet12":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedList/CS/remarks.cs" interactive="try-dotnet-method" id="Snippet12":::
```
Comment (mairaw) [inline xml/System.Collections.Generic/SortedList`2.xml:120]:
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedList/CS/remarks.cs"  id="Snippet12":::
```

File: xml/System.Collections.Generic/SortedList`2.xml [hunk 3]
```diff
@@ -100,10 +100,10 @@
...
   
- [!code-cpp[Generic.SortedList#11](~/samples/snippets/cpp/VS_Snippets_CLR/Generic.SortedList/cpp/remarks.cpp#11)]
- [!code-csharp[Generic.SortedList#11](~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedList/CS/remarks.cs#11)]
- [!code-vb[Generic.SortedList#11](~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.SortedList/VB/remarks.vb#11)]  
-  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/Generic.SortedList/cpp/remarks.cpp" id="Snippet11":::
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedList/CS/remarks.cs" interactive="try-dotnet-method" id="Snippet11":::
```
Comment (mairaw) [inline xml/System.Collections.Generic/SortedList`2.xml:104]:
```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedList/CS/remarks.cs" id="Snippet11":::
```

File: xml/System.Collections.Generic/SortedList`2.xml [hunk 4]
```diff
@@ -100,10 +100,10 @@
...
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.SortedList/VB/remarks.vb" id="Snippet11":::
+
```
Comment (mairaw) [inline xml/System.Collections.Generic/SortedList`2.xml:106]:
```suggestion
  
```

File: xml/System.Collections.Generic/SortedList`2.xml [hunk 5]
```diff
@@ -135,9 +135,9 @@
   
  Finally, the example demonstrates the <xref:System.Collections.Generic.SortedList%602.Remove%2A> method.  
   
- [!code-cpp[Generic.SortedList#1](~/samples/snippets/cpp/VS_Snippets_CLR/Generic.SortedList/cpp/source.cpp#1)]
- [!code-csharp[Generic.SortedList#1](~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedList/CS/source.cs#1)]
- [!code-vb[Generic.SortedList#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.SortedList/VB/source.vb#1)]  
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/Generic.SortedList/cpp/source.cpp" id="Snippet1":::
```
Comment (mairaw) [inline xml/System.Collections.Generic/SortedList`2.xml:138]:
```suggestion
 [!code-cpp[Generic.SortedList#1](~/samples/snippets/cpp/VS_Snippets_CLR/Generic.SortedList/cpp/source.cpp#1)]
```

File: xml/System.Collections.Generic/SortedList`2.xml [hunk 6]
```diff
@@ -135,9 +135,9 @@
...
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedList/CS/source.cs" interactive="try-dotnet" id="Snippet1":::
```
Comment (mairaw) [inline xml/System.Collections.Generic/SortedList`2.xml:139]:
```suggestion
 [!code-csharp[Generic.SortedList#1](~/samples/snippets/csharp/VS_Snippets_CLR/Generic.SortedList/CS/source.cs#1)]
```

File: xml/System.Collections.Generic/SortedList`2.xml [hunk 7]
```diff
@@ -135,9 +135,9 @@
...
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.SortedList/VB/source.vb" id="Snippet1":::
```
Comment (mairaw) [inline xml/System.Collections.Generic/SortedList`2.xml:140]:
```suggestion
 [!code-vb[Generic.SortedList#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/Generic.SortedList/VB/source.vb#1)]  
```
